modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro

```diff
@@ -4,6 +4,24 @@ definition {
 		Click(locator1 = "AppBuilder#BUTTON_ADD");
 	}
 
+	macro addApp {
+		AppBuilderAdmin.switchTab(tabName = "Apps");
+
+		AppBuilderAdmin.add();
+
+		AppBuilderAdmin.selectForm(formName = "${formName}");
```

Comment:
Hey Brooke, could you please rename formName to formViewName and tableName to tableViewName to keep the same pattern used when creating Form/Table Views?